pwndbg/aglib/nearpc.py

```diff
@@ -0,0 +1,355 @@
+from __future__ import annotations
+
+from typing import List
+
+from capstone import *  # noqa: F403
+
+import pwndbg
+import pwndbg.aglib.disasm
+import pwndbg.aglib.regs
+import pwndbg.aglib.strings
+import pwndbg.aglib.vmmap
+import pwndbg.color
+import pwndbg.color.context as C
+import pwndbg.color.disasm as D
+import pwndbg.color.theme
+import pwndbg.commands.comments
+import pwndbg.integration
+import pwndbg.lib.config
+import pwndbg.lib.functions
+import pwndbg.ui
+from pwndbg.aglib.disasm.instruction import SplitType
+from pwndbg.color import ColorConfig
+from pwndbg.color import ColorParamSpec
+from pwndbg.color import message
+
+# We prefix all of our configuration options with a special `aglib-` prefix if
+# we detect that gdblib is present. Pwndbg doesn't allow multiple settings under
+# the same name, so we have to add a unique prefix until we move all of Pwndbg
+# to this module, rather than the gdblib one.
```

Comment:
Not sure I like it :/

Reply:
I don't like, either, but is there a better way to resolve that before we move everything over to `pwndbg.gdblib.nearpc`?